saleor/plugins/webhook/plugin.py

```diff
@@ -429,3 +432,23 @@ def process_payment(
             previous_value,
             **kwargs,
         )
+
+    def get_shipping_methods(
+        self, checkout: Optional["Checkout"], previous_value, **kwargs
+    ) -> List["ExternalShippingMethod"]:
+        methods = []
+        apps = App.objects.for_event_type(
+            WebhookEventType.SHIPPING_LIST_METHODS
+        ).prefetch_related("webhooks")
+        for app in apps:
+            response_data = trigger_webhook_sync(
+                event_type=WebhookEventType.SHIPPING_LIST_METHODS,
+                data=generate_shipping_methods_payload(checkout),
```

Comment:
It is the same checkout, I don't think that we need to re-generate payload  for each app call